hvpy/api_groups/jpeg2000/get_jp2_image.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class getJP2ImageInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the `getJP2Image API.
+
+    <https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image>`__.
+
+    Attributes
+    ----------
+    date : datetime
+        Desired date/time of the JP2 image.
+    sourceId : int
+        Unique image datasource identifier.
+    jpip : bool, optional
+        Returns a JPIP URI instead of the binary data of the image if set to `True`, defaults to `False`.
+    Json : bool, optional
```

Comment:
```suggestion
    json : bool, optional
```
and elsewhere in this pull request.